apps/rush-lib/src/utilities/RushCiMode.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as isCI from 'is-ci';
+import * as colors from 'colors';
+
+/**
+ * Determines if Rush is running in CI mode or NON-CI mode
+ */
+export class RushCiMode {
```

Comment:
I don't love this being static. Can this be attached to `RushConfiguration`?

Reply:
Environment processing in EnvironmentConfiguration and holding the result as readonly in RushConfiguration might not be easy - let me see how RushConfiguration is set up.